Machine Learning/sklearn examples/K Mean Clustering/K-Mean Clustering .ipynb

```diff
@@ -0,0 +1,631 @@
+{
```

Comment:
<p>The notebook is currently raw and has no explanations so as to what is happening, you should include explanations to help people understand.</p><p> </p><p>Please follow these guidelines:</p><p> </p><p>### Include Author name</p><p> </p><p>You would be expected to include under the heading:</p><p> </p><p> </p><p><code>_Originally Contributed by [GitHub-Username](https://github.com/GitHub-Username)_</code></p><p> </p><p> </p><p>### Include buttons for accessibility</p><p>Include a <code>Run in Google Colab</code> and <code>View Source on GitHub</code> buttons</p><p> </p><p>#### Example (taken from the [Linear Regression Notebook](https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Linear%20Regression/Linear%20Regression%20.ipynb) in this repo)-</p><p> </p><p>_Notebook orignially contributed by: [Aditya-1500](https://github.com/Aditya-1500)_</p><p> </p><p>&lt;table class="tfo-notebook-buttons" align="left"&gt;</p><p>&nbsp;&lt;td&gt;</p><p>&nbsp;&nbsp;&lt;a target="_blank" href="https://colab.research.google.com/github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Linear%20Regression/Linear%20Regression%20.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/colab_logo_32px.png" /&gt;Run in Google Colab&lt;/a&gt;</p><p>&nbsp;&lt;/td&gt;</p><p>&nbsp;&lt;td&gt;</p><p>&nbsp;&nbsp;&lt;a target="_blank" href="https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Linear%20Regression/Linear%20Regression%20.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" /&gt;View source on GitHub&lt;/a&gt;</p><p>&nbsp;&lt;/td&gt;</p><p>&lt;/table&gt;</p><p> </p><p>&lt;br&gt;</p><p>&lt;br&gt;</p><p>&lt;br&gt;</p><p> </p><p>#### Code to generate buttons</p><p> </p><p> </p><p><code>&lt;table class="tfo-notebook-buttons" align="left"&gt;</code></p><p><code>&nbsp;&lt;td&gt;</code></p><p><code>&nbsp;&nbsp;&lt;a target="_blank" href="https://colab.research.google.com/github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Logistic%20Regression/Logistic%20Regression.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/colab_logo_32px.png" /&gt;Run in Google Colab&lt;/a&gt;</code></p><p><code>&nbsp;&lt;/td&gt;</code></p><p><code>&nbsp;&lt;td&gt;</code></p><p><code>&nbsp;&nbsp;&lt;a target="_blank" href="https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Logistic%20Regression/Logistic%20Regression.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" /&gt;View source on GitHub&lt;/a&gt;</code></p><p><code>&nbsp;&lt;/td&gt;</code></p><p><code>&lt;/table&gt;</code></p><p> </p><p>### Include a short intro</p><p> </p><p>Below the above-mentioned notebook buttons, be sure to add a small intro showing what the notebook does.</p><p> </p><p>### Include clear Dataset Credits</p><p> </p><p>In the notebook, intro be sure to add clear credits to the dataset owner.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
hii
@Rishit-dagli I have made the necessary changes in the notebook but should I delete the older notebook and upload new one and create new pull request?

Reply:
@aditya9110 Just make these changes in the [master branch of your fork](https://github.com/aditya9110/Python3-by-practice) and they will appear in this PR since you are comparing from [aditya9110:master](https://github.com/aditya9110/Python3-by-practice)

Reply:
okay got it